Sources/ClientRuntime/Networking/AwsChunked/AwsChunkedBufferedStream.swift

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AwsCommonRuntimeKit
+
+class AwsChunkedBufferedStream {
```

Comment:
BufferedStream and FileStream are also in smithy-swift. I figured adding the chunked classes here would make sense since a lot of these concepts arent specific to aws (despite the name).